guides/doc-Managing_Security_Compliance/master.adoc

```diff
@@ -6,7 +6,7 @@ include::common/header.adoc[]
 = {ManagingSecurityDocTitle}
 
 ifdef::foreman-deb[]
-This feature is not available for Debian/Ubuntu.
+This feature is not available for hosts running {DL}.
```

Comment:
```suggestion
This feature is not available for {Project} on {DL}.
```

I checked via `rg -i scap` in foreman-packaging on deb/develop. This means that the comment is true and you cannot use this feature if Foreman runs on Debian/Ubuntu because we do not package the foreman_openscap plugin.